### ISSUE TITLE
Create endpoint to request datapoints for an area

### ISSUE THREAD

**Issue by Superschnizel** [amosproj/amos2024ss04-building-information-enhancer#131]
## Description
Implement the Endpoint `/datasets/1/data`
with the parameters 
bottomLat, bottomLong, topLat, topLong, zoomLevel

it should return all datapoints from dataset 1 that are inside the area given by the top and bottom points. return format is geojson.

## Acceptance criteria
* Api endpoints returns data.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
